src/CommunityToolkit.Maui.Core/Essentials/Badge/BadgeImplementation.tizen.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Tizen.Applications;
+using Tizen.NUI;
+
+namespace CommunityToolkit.Maui.ApplicationModel;
+
+/// <inheritdoc />
+public class BadgeImplementation : IBadge
+{
+	/// <inheritdoc />
+	public void SetCount(uint count)
+	{
+		var appId = Application.Current.ApplicationInfo.ApplicationId;
+		if (count <= 0)
```

Comment:
it can't be less than zero
```suggestion
		if (count is 0)
```